src/styles/index.scss

```diff
@@ -1,3 +1,153 @@
+$font-family-primary: 'Roboto', Arial, sans-serif;
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: 400px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: 166px;
+    margin-inline: 16px;
```

Comment:
instead of hardcoding width and setting margin-inline for each element add padding inline to the entire card